executor/adapter.go

```diff
@@ -358,8 +358,16 @@ func (a *ExecStmt) buildExecutor(ctx context.Context) (Executor, error) {
 }
 
 func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
+	level := log.GetLevel()
+	if level < log.WarnLevel {
+		return
+	}
 	cfg := config.GetGlobalConfig()
 	costTime := time.Since(a.startTime)
+	threshold := time.Duration(cfg.Log.SlowThreshold) * time.Millisecond
+	if costTime < threshold && level < log.DebugLevel {
```

Comment:
ditto

Reply:
If cost time is less than threshold, and log level is not debug, do not print query log. This modification keeps the same logic as before.